_includes/header.html

```diff
@@ -1,217 +1,181 @@
-<div class="syn-styles-supported syn-header-wrapper">
-  <header class="syn-page-header">
-    <div class="syn-page-header-nav-backdrop" onclick="['nav-open'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))"></div>
-    <div class="syn-page-header-content syn-flex justify-space-between">
-      <div class="syn-flex align-center">
-        <button
-          class="syn-btn icon text grow syn-hidden-lg"
-          onclick="['nav-open', 'nav-collapsed'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))"
-        >
-          <svg width="16px" height="16px" viewBox="0 0 12 12">
-            <path
-              fill="currentColor"
-              d="M0,2 L12,2 L12,3.344 L0,3.344 L0,2 Z M0,6.67 L0,5.33 L12,5.33 L12,6.67 L0,6.67 Z M0,10 L0,8.656 L12,8.656 L12,10 L0,10 Z"
-            />
-          </svg>
-        </button>
-        <a class="syn-flex" href="/">
-          {% if site.isLocalDev or site.enterprise %}
-          <img
-            class="syn-ml-8"
-            src="{{ site.baseurl }}/images/algo-purple.svg"
-            alt="Algorithmia Logo"
-            style="max-height: 24px; max-width: 100%"
-          />
-          {% else %}
-          <img
-            class="syn-ml-8"
-            src="/v1/config/logo"
-            alt="Algorithmia Logo"
-            style="max-height: 24px; max-width: 100%"
-          />
-          {% endif %}
+<header class="syn-styles-supported supports-synapse-typography syn-page-header">
+  <div class="utility-header syn-visible-lg">
+    <div class="syn-pr-32 syn-pv-8 syn-flex flex-row justify-end">
+      <div class="hide-if-signed-in">
+        <a href="https://teams.algorithmia.com/signin" data-e2e="header-signin-link" class="syn-link syn-font-weight-bold syn-text-secondary syn-text-sm">
```

Comment:
Let's keep this as `/signin` for now. That way behavior is controlled by golduck's routes file and isn't contained across several different repos. We lose some efficiency due to the redirect but maintenance is significantly easier.
```suggestion
        <a href="/signin" class="syn-link syn-font-weight-bold syn-text-secondary syn-text-sm">
```